doc/source/whatsnew/v0.25.0.rst

```diff
@@ -214,7 +214,7 @@ I/O
 - Bug in :func:`read_json` for ``orient='table'`` when it tries to infer dtypes by default, which is not applicable as dtypes are already defined in the JSON schema (:issue:`21345`)
 - Bug in :func:`read_json` for ``orient='table'`` and float index, as it infers index dtype by default, which is not applicable because index dtype is already defined in the JSON schema (:issue:`25433`)
 - Bug in :func:`read_json` for ``orient='table'`` and string of float column names, as it makes a column name type conversion to Timestamp, which is not applicable because column names are already defined in the JSON schema (:issue:`25435`)
--
+- Bug in :func:`json_normalize` for ``errors='ignore'`` and missing metadata fields, the missing metadata values in dataframe were filled with the string "nan" instead of ``numpy.nan`` (:issue:`25468`)
```

Comment:
more like 

Bug in :func:`json_normalize` for ``errors='ignore'`` where missing values in the input data, were filled in resultin DataFrame with the string "nan" instead of ``numpy.nan`` 

make sure to use double backticks on DataFrame